app/src/main/java/org/sopt/official/feature/home/HomeViewModel.kt

```diff
@@ -36,15 +39,19 @@ import javax.inject.Inject
 
 @HiltViewModel
 class HomeViewModel @Inject constructor(
-    private val mainViewRepository: HomeRepository
+    private val mainViewRepository: HomeRepository,
+    private val registerPushTokenUseCase: RegisterPushTokenUseCase,
 ) : ViewModel() {
+
     private val homeUiState = MutableStateFlow(SoptUser())
     private val user = homeUiState.map { it.user }
+
     private val _description = MutableStateFlow(HomeSection())
     val description = _description.asStateFlow()
     val userActiveState = homeUiState
         .map { it.user.activeState }
         .stateIn(viewModelScope, SharingStarted.WhileSubscribed(), UserActiveState.UNAUTHENTICATED)
+    val isAllNotificationsConfirm: Flow<Boolean> = homeUiState.map { it.isAllConfirm }
```

Comment:
```suggestion
    val isAllNotificationsConfirm = homeUiState.map { it.isAllConfirm }
```

type 없애도 될 것 같습니다...!!